codenteq/interfeys
# React version 18

## Issue Reference


## Description
With the release of React v19, some issues in the Interfeys design system were discovered. To address these compatibility issues and ensure stability, the system has been reverted back to React v18.

## How To Test This?
1. Pull the project and switch to the relevant branch.
2. Update dependencies by running `npm install`.
3. Run the project and test for any crashes or issues in the UI.
4. Verify the updates are compatible with React v18 and compare with previous versions.

## Documentation
The documentation should be updated to reflect the change from React v19 back to React v18 due to compatibility issues.
